BO4E/BO/Berechnungsformel.cs

```diff
@@ -0,0 +1,74 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Text.Json.Serialization;
+using BO4E.COM;
+using BO4E.ENUM;
+using BO4E.meta;
+using Newtonsoft.Json;
+using ProtoBuf;
+using Verwendungszweck = BO4E.COM.Verwendungszweck;
+
+namespace BO4E.BO
+{
+    /// <summary>
+    /// Eine Berechnungsformel beschreibt, wie komplexe MaLo-MeLo-Konstrukte rechnerisch behandelt werden.
+    /// </summary>
+    /// <remarks>Berechnungsformeln werden in der Marktkommunikation mit Prüfidentifikator 25001 (UTILTS) übermittelt</remarks>
+    [ProtoContract]
+    public class Berechnungsformel : BusinessObject
+    {
+        // lieferrichtung is part of the marktlokations-bo
+
+        [System.Text.Json.Serialization.JsonIgnore]
+        [Newtonsoft.Json.JsonIgnore]
+        [ProtoMember(6, Name = nameof(Beginndatum))]
+        [CompatibilityLevel(CompatibilityLevel.Level240)]
+        private DateTime _Beginndatum
+        {
+            get => Beginndatum.UtcDateTime;
+            set => Beginndatum = DateTime.SpecifyKind(value, DateTimeKind.Utc);
+        }
+        /// <summary>
+        /// Der inklusive Zeitpunkt ab dem die Berechnungsformel gültig ist
+        /// </summary>
+        [JsonProperty(Required = Required.Always, Order = 5, PropertyName = "beginndatum")]
+        [JsonPropertyName("beginndatum")]
+        [ProtoIgnore]
+        [DataCategory(DataCategory.FINANCE)]
```

Comment:
Warum ist das Beginndatum FINANCE?

Reply:
habe die datacategory entfernt. die wurden in letzter zeit eh kaum mehr gepflegt. wurden ursprünglich zur anonymisieren verwendet. da konnte man dann z.B. sagen: "ich will alle personenbezogenen/alle finanziellen Daten löschen" und dann wurden die jeweiligen Properties gehasht oder gelöscht.
wird aber gerade nicht aktiv genutzt.

generell sollten wir mal aufräumen. wir schleppen in jedem bo und in jeder property so viel overhead mit rum (der ganze proto-kram, die BoKeys, diese datacategories...) ich weiß nur nicht, welcher service was verwendet und braucht

Reply:
7456b407754e22977faf6ddf330f1cc7125c4f23